Mlem/Views/Tabs/Inbox/Inbox View.swift

```diff
@@ -146,26 +195,17 @@ struct InboxView: View {
                         MessagesFeedView(messageTracker: messageTracker)
                     }
                 }
-            }
-            .fancyTabScrollCompatible()
-            .refreshable {
-                // wrapping in task so view redraws don't cancel
-                // awaiting the value makes the refreshable indicator properly wait for the call to finish
-                await Task {
-                    await refresh()
-                }.value
-            }
-            .hoistNavigation {
-                withAnimation {
-                    scrollProxy.scrollTo(scrollToTop)
+            } header: {
+                VStack(spacing: 0) {
+                    BubblePicker(InboxTab.allCases, selected: $curTab) { tab in
```

Comment:
The vertical padding on this view doesn't match the vertical padding we use in `CommunityFeedView`. We could make the padding a little taller to match?